app/helpers/pages_helper.rb

```diff
@@ -1,7 +1,7 @@
 module PagesHelper
   def product_button(product)
     if signed_in?
-      render(partial: 'product_button', locals: { product: product })
+      render(partial: 'shared/product_button', locals: { product: product })
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.